NeoSwift/Account.swift

```diff
@@ -170,21 +169,21 @@ public class Account {
         
         if needsTwoOutputTransactions {
             //Transaction To Reciever
-            payload = payload + [0x02] + asset.rawValue.dataWithHexString().bytes.reversed()
+            payload = payload + [0x02] + Array<UInt8>(asset.rawValue.dataWithHexString()).reversed()
```

Comment:
Let's add the .bytes extension as well, looking at this ```data.bytes``` reads a lot easier than Array<UInt8>(data) especially since we want to avoid long form Array declarations